cpu/arm7_common/Makefile.features

```diff
@@ -1,5 +1,5 @@
-CPU_ARCH = armv4t
-CPU_CORE = arm7tdmi_s
+CPU_ARCH := armv4t
+CPU_CORE := arm7tdmi_s
```

Comment:
Do you mind putting these changes in its own commit, they are not really related to the whole PR. And you can add the argument for doing this to the commit message.